src/main/java/pages/accounts/CreateNewAccountsPage.java

```diff
@@ -0,0 +1,127 @@
+package pages.accounts;
+
+import base.BasePage;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import pages.accounts.AccountsPage;
+
+public class CreateNewAccountsPage extends BasePage {
+
+    @FindBy(xpath = "//span[text() = 'Account Name']/../..//input[@type = 'text']")
+    private WebElement accountNameField;
+
+    @FindBy(xpath = "//button[@title = 'Save']")
+    private WebElement saveNewCcountButton;
+
+    @FindBy(xpath = "//span[text() = 'Active']/../..//div[@class = 'uiMenu']")
+    private WebElement activeDropDown;
+
+    @FindBy (xpath = "//div[@class = 'select-options']//a[text() = 'Yes']")
+    private WebElement chooseActiveinActiveDropDown;
+
+    @FindBy(xpath = "//span[text() = 'Date Picker']")
+    private WebElement slaExpirationDateDropDownCalendar;
+    @FindBy(xpath = "//td[@class = 'slds-is-today slds-is-selected uiDayInMonthCell']")
+    private WebElement todaysDateInSlaExpirationDateDropDownCalendar;
+
+    @FindBy(xpath = "//span[text() = 'SLA']/../..//div[@class = 'uiMenu']")
+    private WebElement slaDropDown;
+    @FindBy (xpath = "/html/body/div[8]/div/ul/li[2]/a")
```

Comment:
Add empty line between these lines and change locator's path to more understandable.

Reply:
done